docs/CI.md

```diff
@@ -23,22 +23,18 @@ Honeycomb includes workflows to build and create installers for Windows. Mac and
 
 
 More specifically, the following workflows are included:
-* `build-clinic-win.yaml`: Every time an Pull Request is open, or a push is made to the `main` branch, the software is built and tests are run (clinic version). This workflow does not build and upload desktop installers
-* `build-home-win.yaml`: Every time an Pull Request is open, or a push is made to the `main` branch, the software is built and tests are run (home version). his workflow does not build and upload desktop installers
-* `package-home-win.yaml`: Create home-installers for windows on demand <sup>1</sup>. The installer is uploaded as an artifact and it is available for download from the GitHub Actions tab
-* `package-home-all.yml`: Create home-installers for all platforms on demand <sup>1</sup>. This workflow also builds the PsiTurk version. The installers are uploaded as an artifact and they are available for download from the GitHub Actions tab.
-* `package-clinic-all.yml`: Create clinic-installers for all platforms on demand <sup>1</sup>. The installers are uploaded as an artifact and they are available for download from the GitHub Actions tab.
-* `package-clinic-win.yaml`: Create clinic-installers for Windows on demand <sup>1</sup>. The installer is uploaded as an artifact and it is available for download from the GitHub Actions tab
-* `release-home-all.yml`: Every time an a Tag is released, home-installers are created and uploaded as packages. This also builds PsiTurk version, and deploys to GitHub pages a web version of the application. 
-* `release-clinic-all.yml`:Every time an a Tag is released, clinic-installers are created and uploaded as packages
+* `build.yaml`: Every time an Pull Request is open, or a push is made to the `main` branch, the software is built and tests are run for all platforms and `at-home` and `clinic` configurations. This workflow does not build and upload desktop installers
+* `package.yaml`: Create installers for all platforms and `at-home` and `clinic` configurations on demand <sup>1</sup>. The installer/executable is uploaded as an artifact and it is available for download from the GitHub Actions tab. This also builds PsiTurk version, and deploys to GitHub pages a web version of the application. 
```

Comment:
`for all platforms and `at-home` and `clinic` configurations` => `for any/all platforms for the `home` or `clinic` setting`

Reply:
`This also builds PsiTurk version` => `This also builds PsiTurk version when `linux` or `all` operating systems are selected`

Reply:
package.yml doesn't deploy to gh-pages